packages/core/src/tracing/cls.js

```diff
@@ -48,6 +50,7 @@ function init(config, _processIdentityProvider) {
     serviceName = config.serviceName;
   }
   processIdentityProvider = _processIdentityProvider;
+  allowRootExitSpan = process.env.INSTANA_ALLOW_ROOT_EXIT_SPAN === '1' || config?.tracing?.allowRootExitSpan;
```

Comment:
This variable is introduced here because the config is already imported in here as a param. So there is no need to import it again